ConTabs/Column.cs

```diff
@@ -54,21 +54,6 @@ public class Column
         /// A List of the values stored
         /// </summary>
         public List<Object> Values { get; set; }
-        public int MaxWidth
```

Comment:
Does removing this mean that the table will always fill the full width provided to it? I think it would be better if we retained the default behaviour of taking as much room as needed, for two reasons:

1) If we don't need the full width of the console, we shouldn't stretch to use it.

2) Sometimes we will want to just render a table without having to know what width we want it to be.

The advantage of the current behaviour is that it's really simple to knock out a quick table in almost any environment. Being able to set a max width (or having some method work it out from the console window) is great, but it should _add to_ rather than _replace_ the default behaviour.

Reply:
I seem to remember that the behavior stayed the same, only difference is that this calculation is being used only if you use some of the stretch styles. You can see that the default behavior stayed unchanged for example from tests - even when they are using the long strings, the outputted tables are still the same than before my changes. I was thinking about backwards compatibility, but I could not think of real case scenario where it could have become a problem. But if you disagree, I am willing to think it through.